src/borg/crypto/key.py

```diff
@@ -749,6 +749,18 @@ class AuthenticatedKey(ID_BLAKE2b_256, RepoKey):
     ARG_NAME = 'authenticated'
     STORAGE = KeyBlobStorage.REPO
 
+    # It's only authenticated, not encrypted.
```

Comment:
I changed the variable name to something that is perhaps more clear without further context, and also added some documentation (also on some other members).